src/CRM/CivixBundle/Resources/views/Code/test-e2e.php.php

```diff
@@ -30,7 +30,7 @@
  */
 class <?php echo $testClass ?> extends \PHPUnit\Framework\TestCase implements EndToEndInterface {
 
-  public static function setUpBeforeClass() {
+  static public function setUpBeforeClass() {
```

Comment:
My understanding is visibility first

Reply:
IDE seems fairly willing to back up it's claim...

<img width="572" alt="Screen Shot 2020-07-03 at 11 07 56 AM" src="https://user-images.githubusercontent.com/336308/86416220-7867ef00-bd1d-11ea-8f27-deec2f69a387.png">

Reply:
hang on you are right...

Reply:
I reversed the wrong ones

Reply:
fixed